articles/references/concepts/stereotypes.md

```diff
@@ -0,0 +1,76 @@
+# Stereotypes
+
+## What is a Stereotype?
+
+>Stereotype is a profile class which defines how an existing metaclass may be extended as part of a profile. It enables the use of a platform or domain specific terminology or notation in place of, or in addition to, the ones used for the extended metaclass.
+
+_from: [uml-diagrams.org](https://www.uml-diagrams.org/stereotype.html)_
+
+According to the quote from the UML Diagrams website, a stereotype defines in UML terms that a given Class will have additional meta-data (in a simplified way).
+
+Intent Architect has adopted this notation called `Stereotype` in order to apply additional meta-data on-top of existing Elements.
+
+## What does Stereotypes look like?
+
+Stereotypes are quite flexible in how they appear. You can have a visible indicator on the Element that you work on or it might be only a field where you need to supply additional information in the form of a text field or the like.
+
+![Visible stereotype](images/stereotypes/visible-stereotype.png)
```

Comment:
I think we should move away from using the down arrows in these pictures, it looks like it could be a UI element for collapsing the element. Perhaps we have a letter block, or something that really just looks like an icon from font awesome.